using-timescaledb/continuous-aggregates.md

```diff
@@ -16,145 +17,202 @@ every refresh. Only new and/or invalidated data will be calculated.  Since this
 something like:
 ```sql
-SELECT * FROM device_summary
-WHERE metric_spread > 1800
+SELECT * FROM conditions_summary_daily
+WHERE max - min > 1800
   AND bucket >= '2018-01-01' AND bucket < '2018-04-01'
 ORDER BY bucket DESC, device_id DESC LIMIT 20;
 ```
 
-**How Far Behind Will Continuous Aggregates Be?** [](how-far-behind)  
-When designing continuous aggregates, we wanted to avoid write amplification as
-that could drastically affect insert rate for write-heavy time-series workloads.
-We also wanted to deal gracefully with out-of-order data. There are several
-tunable parameters that allow us to control how quickly data is materialized
-after insert and how far behind the aggregates will remain (so as not to cause
-the same materialization to be modified multiple times and cause write amplification).
-
-We usually stay at least one `bucket_width` (the first argument to the
-`time_bucket` call in the view definition) behind the current time.
-The `refresh_lag` parameter determines how much further behind the
-current time the background worker will attempt to materialize
-(we will be behind by `refresh_lag` + `bucket_width`).  So in our example, if
-the `refresh_lag` is 1 hour and the `bucket_width` is 1 hour, the
-materialization will generally be 2 hours behind the current time.
-Tuning the `refresh_lag` parameter lower will mean that the
-aggregates will follow inserts more closely, but can cause some write
-amplification which may degrade insert performance. Setting `refresh_lag` to
-`-<bucket_width>` will keep the continuous aggregate up-to-date.
-
-The `refresh_interval` controls how frequently materialization jobs will be
-launched. Setting a shorter interval will mean materializations happen more
-"eagerly" but can consume background worker resources while it is running.
-
-The `timescaledb.max_interval_per_job` parameter can also be useful, especially
-when a continuous aggregate is created on a hypertable that already contains
-data and the materialization job needs to catch up. It determines the maximum
-amount of data processed in a single job. If there is still more work to do to
-catch up to where we need to be based on the `refresh_lag` setting, another job
-will be spawned immediately following completion. This means that the part of the
-data processed by previous jobs will be available even before the
-materialization is fully caught up.
+### Advanced Usage
+
+The `timescaledb.refresh_interval` option controls how frequently
+materialization jobs will be launched. Setting a shorter interval will
+mean materializations happen more frequently but can consume
+background worker resources while it is running.
+
+When querying the continuous aggregate view (for example,
+`conditions_summary_daily`) you want to get a complete view of the
+data including both the partially aggregated data the data in the
+underlying table (for example, `conditions`). For this reason, the
```

Comment:
You can remove the last sentence with the above changes.

Reply:
Removed.